### PR TITLE
proposed update for required section

### PR DESCRIPTION
**Pull request by sunstonesecure-robert**
the project asked to explicitly tag the recommendations section to meet incubation requirements for their presentations. None of these are curated "GHI-ready" requests, but more high level best practice suggestions.

Edits include renaming title to "TAG Recommendation" - and some small edits and clarifications of the bulleted items.

@ashutosh-narkar

### CODE REVIEW

**Comment by sunstonesecure-robert** [comment]
not sure how to override spelling since it's not a mistake, it's a project name (sigstore)?

File: assessments/projects/custodian/joint-review.md [hunk 1]
```diff
@@ -859,14 +858,15 @@ stale since the only In Progress item was last updated Jan 22, 2020.
...
   python and YAML files should be controlled using cryptographic mechanisms for
   integrity. As this is an area of active research in the community, there are
   many possible tools to address the supply chain and distribution of the c7n
-  tool and docker image.
+  tool and docker image securely.
 * The c7n code should itself implement some sort of YAML policy integrity and,
   possibly, efficacy/safety check(s). This may prevent casual tampering of YAML
-  policies once deployed in an organization.
+  policies once deployed in an organization. For example signing and signature
+  check with cosign/sigstore or [JWT](https://github.com/open-policy-agent/opa/issues/1757).
```
Comment (lumjjb) [inline assessments/projects/custodian/joint-review.md:869]:
```suggestion
  check with Cosign/Sigstore or [JWT](https://github.com/open-policy-agent/opa/issues/1757).
```

Comment (lumjjb) [reply assessments/projects/custodian/joint-review.md:869]: the caps versions are in the spelling ignore, going to try this.

**Comment by lumjjb** [comment]
spell err is gone, looks like mainly just a bad anchor link, and DCO, can you change those and then ill merge it!

**Comment by lumjjb** [comment]
@sunstonesecure-robert any chance you can make the changes to the PR? Just minor DCO and link fixes!

**Review by anvega — APPROVED** [review]

**Pull request merged**